cmake/modules/CompileWarnings.cmake

```diff
@@ -15,14 +15,7 @@ macro(wpilib_target_warnings target)
     else()
         target_compile_options(
             ${target}
-            PRIVATE
-                /wd4146
-                /wd4244
-                /wd4251
-                /wd4267
-                /WX
-                /D_CRT_SECURE_NO_WARNINGS
-                ${WPILIB_TARGET_WARNINGS}
```

Comment:
Do we want to remove the option of customize target warnings?  That seems like a nice feature to retain.

Reply:
This still changes behavior.  We want the custom settings at the end of the option list.